tsrelay/handler/get_peers.go

```diff
@@ -130,7 +131,10 @@ func (h *handler) getPeers(ctx context.Context, body io.Reader) (*getPeersRespon
 			SSHEnabled:   len(p.SSH_HostKeys) > 0,
 			Address:      addr,
 		}
-		if p.UserID == st.Self.UserID {
+
+		if !p.Online {
+			s.PeerGroups[2].Peers = append(s.PeerGroups[2].Peers, peer)
+		} else if p.UserID == st.Self.UserID {
```

Comment:
We'll need to adjust lines 144-151 to include this new group, or make the UI side check the length of each group and not the length of the outer array: https://github.com/tailscale-dev/vscode-tailscale/blob/e30851c76cea533b6645a43377cec577f93e9aa5/src/node-explorer-provider.ts#L186